rubin_sim/maf/plots/skyproj_plotters.py

```diff
@@ -0,0 +1,523 @@
+            `skyproj._Skyproj.draw_polygon`.
+        """
+        galactic_pole = SkyCoord(l=0 * u.degree, b=90 * u.degree, frame="galactic").icrs
+        self.draw_circle(galactic_pole.ra.deg, galactic_pole.dec.deg, **self.galactic_plane_kwargs)
+
+    def draw_zd(self, zd=90, **kwargs):
+        """Draw a circle at a given zenith distance.
+
+        Parameters
+        ----------
+        zd : `float`
+            The zenith distance to draw, in degrees.
+            Defaults to 90.
+        **kwargs
+            Keyword arguments passed to
+            `skyproj._Skyproj.draw_polygon`.
+
+        Notes
+        -----
+        This method relies on the site location and time (as an mjd) set in
+        the ``model_observatory`` element of the ``plot_dict``, which
+        should be of the class
+        `rubin_scheduler.scheduler.model_observatory.ModelObservatory`.
+        """
+        if "model_observatory" not in self.plot_dict or self.plot_dict["model_observatory"] is None:
+            warnings.warn("plot_dict['model_observatory'] must be set to plot a zenith distance circle")
+            return
+
+        model_observatory = self.plot_dict["model_observatory"]
+        lmst = model_observatory.return_conditions().lmst * 360 / 24
+        latitude = model_observatory.location.lat.deg
+        self.draw_circle(lmst, latitude, zd, **kwargs)
+
+    def draw_body(self, body="sun", **kwargs):
+        """Mark the sun or moon.
+
+        Parameters
+        ----------
+        body : `str`
+            The name of the body to draw, either `sun` or `moon`.
+            Defaults to `sun`
+        **kwargs
+            Keyword arguments passed to
+            `skyproj._Skyproj.scatter`
+
+        Notes
+        -----
+        This method relies on the site location and time (as an mjd) set in
+        the ``model_observatory`` element of the ``plot_dict``, which
+        should be of the class
+        `rubin_scheduler.scheduler.model_observatory.ModelObservatory`.
+        """
+        if "model_observatory" not in self.plot_dict or self.plot_dict["model_observatory"] is None:
+            warnings.warn(f"plot_dict['model_observatory'] must be set to plot the {body}")
+            return
+
+        mjd = self.plot_dict["model_observatory"].mjd
+        model_observatory = self.plot_dict["model_observatory"]
+        sun_moon_positions = model_observatory.almanac.get_sun_moon_positions(mjd)
+        ra = np.degrees(sun_moon_positions[f"{body}_RA"].item())
+        decl = np.degrees(sun_moon_positions[f"{body}_dec"].item())
+        self.skyproj.scatter(ra, decl, **kwargs)
+
+    def decorate(self):
+        """Add decorations/annotations to the sky plot.
+
+        Notes
+        -----
+        This method relies on the site location and time (as an mjd) set in
+        the ``model_observatory`` element of the ``plot_dict``, which
+        should be of the class
+        `rubin_scheduler.scheduler.model_observatory.ModelObservatory`.
```

Comment:
It would be great if you did not need ModelObservatory to add the ecliptic and galactic plane.

Reply:
It seems like maybe you don't, and in fact, if you don't add ModelObservatory to the plotdict, then this just works as expected - adding ecliptic and galactic plane, but not the sun/moon?

Reply:
Indeed, ModelObservatory is only needed for decorations that actually need it. I've expanded the comment to make this clearer.